packages/angular-test/src/boolean.ts

```diff
@@ -279,22 +279,26 @@ export const booleanErrorTest = <C extends JsonFormsControl, I>(
   it('should display errors', () => {
     component.uischema = uischema;
 
-    getJsonFormsService(component).init({
+    const formsService = getJsonFormsService(component);
+    formsService.init({
       core: {
         data,
         schema: defaultBooleanTestSchema,
-        errors: [
-          {
-            dataPath: 'foo',
-            message: 'Hi, this is me, test error!',
-            keyword: '',
-            params: '',
-            schemaPath: ''
-          }
-        ],
         uischema: undefined
       }
     });
+    // tslint:disable-next-line: no-string-literal
+    formsService['_state'].core.errors = [
+      {
+        dataPath: 'foo',
+        message: 'Hi, this is me, test error!',
+        keyword: '',
+        schemaPath: '',
+        params: ''
+      }
+    ];
+    // tslint:disable-next-line: no-string-literal
+    formsService['updateSubject']();
```

Comment:
Why do we have to do it like this, couldn't we also use the `setErrors` action?

Reply:
where is the setErrors action defined?

Reply:
It's just an action for the core reducer.

Reply:
Another thing: Don't we have `refresh` defined on the forms service. Couldn't we use that instead of `['updateSubject']`.

Reply:
I'll check